runtime/devnet/src/config/proxy.rs

```diff
@@ -1,4 +1,4 @@
-use super::assets::TrustBackedAssetsCall;
+use super::assets::AssetsCall;
```

Comment:
Appreciate you wanted to simplify this, but it really just does create noise in a large PR.

Reply:
This was from a while back, overlooked this when I tried simplifying the PR, apologies.